src/java/main/org/apache/jute/Utils.java

```diff
@@ -190,19 +190,32 @@ static String fromCSVString(String s) throws IOException {
     }
     
     /**
+     * convert byte array to a string in hex format
      * 
-     * @param s 
-     * @return 
+     * @param barr
+     * @return
      */
-    static String toXMLBuffer(byte barr[]) {
```

Comment:
I went through the links that you provided. Would you mind including the version of the JVM that you are running? According to https://issues.apache.org/jira/browse/CAMEL-4893 the issue I think you are running into was fixed in JDK's greater than 1.7.0_02.

It would also be great if you could include a stack trace of the issue you are running into.